### DISCUSSION TITLE
Cannot access Teleport via Github

### DISCUSSION THREAD

**Discussion by Chuyentest123** [gravitational/teleport#28247] [Q&A] [answered]
Hope you answer my issue soon.

I just do as the document but I can't access Teleport via GitHub
Here is my config file at Teleport and GitHub.yaml

![image](https://github.com/gravitational/teleport/assets/128662730/faa4f8bd-a033-432b-8e8e-39a57f009c48)

![image](https://github.com/gravitational/teleport/assets/128662730/334528e2-baed-4edb-9cef-7ff508f8a084)

And here is my organization and my Team:

![image](https://github.com/gravitational/teleport/assets/128662730/09e66bd6-2ce6-489c-aa99-ee540b95f46a)

![image](https://github.com/gravitational/teleport/assets/128662730/21028298-1fba-4f17-8f2e-fe4392962a61)

![image](https://github.com/gravitational/teleport/assets/128662730/4c4a6858-5cf9-4412-9f8f-48d933909ec5)

And the OAuthApps:

![image](https://github.com/gravitational/teleport/assets/128662730/cc545d92-bb77-44d5-a93b-cb4906b8bbd5)

Here is the progress of login:

This is Chuyentest123's GitHub account (Administrator of the organization and member of TestEditor):

![image](https://github.com/gravitational/teleport/assets/128662730/cef27042-beb1-4984-87ea-713acba37310)

After login GitHub success I have the notification like this:

![image](https://github.com/gravitational/teleport/assets/128662730/29a2aec4-ee7b-4f4f-9ea9-923b51dfa0c8)

Here is the log of the failed login:

![image](https://github.com/gravitational/teleport/assets/128662730/42fc0a33-282b-4492-a5a1-754708efaf55)

And here are the logs from the Teleport process itself. Seems like it tells me that I have some typo in the config but I don't.

![image](https://github.com/gravitational/teleport/assets/128662730/a1a2b6da-c6bf-46ac-8723-72a86daec2b3)

Teleport version: 11.4.4

**Comment by dgkirkwood** [comment, marked as answer]
Hi @Chuyentest123 -

It appears the attribute coming through in the SSO Login Failed message has your organisation as `testTeleport` (Notice the capital on Teleport).

I can also see this capitalisation in your screenshots for your org. 

Could you try updating your connector configuration to reflect this? 

`organization: testTeleport`

Then test a login again and see if it is succesful.

Thanks,
Dan.

**Reply by webvictim** [reply]
FYI this is a duplicate of https://github.com/gravitational/teleport/discussions/27263 where I _think_ we tried that already.

**Reply by Chuyentest123** [reply]
Thank you for your help. It's really working.
